client/view/base/index.css

```diff
@@ -1,7 +1,83 @@
-@import "./reset.css";
+@import "./BrowsersStat.css";
+@import "./Article.css";
+@import "./Form.css";
+@import "./Introduction.css";
+@import "./Badge.css";
+@import "./QueryLinks.css";
 
 @media (prefers-reduced-motion: reduce) {
   * {
     transition: none !important;
   }
 }
+
+body {
+  display: flex;
+  height: 100vh;
+  margin: 0;
+  font-family:
```

Comment:
Let’s use `system-ui` font and polyfill (if we still need it)

Reply:
What it is?... Can you give me a link?

BTW, we wiil use a custom font, according the figma, after MVP.

Reply:
> system-ui
> Glyphs are taken from the default user interface font on a given platform. Because typographic traditions vary widely across the world, this generic is provided for typefaces that don't map cleanly into the other generics.


https://developer.mozilla.org/ru/docs/Web/CSS/font-family

https://caniuse.com/?search=system-ui

Reply:
https://infinnie.github.io/blog/2017/systemui.html

Reply:
Hm, interesting problem. OK, let’s keep allow-list and think about it later.